src/components/SelectChildForClass.tsx

```diff
@@ -20,12 +28,8 @@ export default function SelectChildForClass(
                 </Text>
             </Center>
             <Center>
-                <Text pb="55px" align="center" mt="30px">
-                    Who would you like to register for{" "}
-                    <b>
-                        Building Bridges with Music - <br />
-                        Singing Monkeys (Ages 9 and under)?{" "}
-                    </b>
+                <Text pb="55px" align="center" mt="30px" width="60%">
+                    Who would you like to register for <b>{props.className}</b>
```

Comment:
I think you also need the program name here. Also, please rebase and use i18n for translation integration:
```suggestion
                    {t("enroll.selectChild")} <b>{props.programName} - {props.className} ({t(
                                    classInfo.isAgeMinimal
                                        ? "program.ageGroupAbove"
                                        : "program.ageGroupUnder",
                                    {
                                        age: classInfo.borderAge,
                                    },
                                )})</b>
```
we can ignore the `(select one)` for now. 

^ where t is the object in the useTranslation hook for the form translations (should be in the file once you rebase)

Reply:
@MatoPlus i already did that in the other file [here](https://github.com/uwblueprint/social-diversity-for-children/pull/145/files#diff-83ff1da87167756ae59b558afa4424bea36de0102fedd2a29dc2a725e93c1608R113) - lines 113-121 ; will rebase for the `enroll.selectChild` part